From d75b4c56c2221fc365ea50e421f877463e8b3542 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Mon, 5 Jan 2026 19:34:32 +0100
Subject: [PATCH 029/N] python-config: use the Python variant of
 python-config

so there is no bash required to run it on Windows
---
 Makefile.pre.in | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Makefile.pre.in b/Makefile.pre.in
index 258e500..41118d5 100644
--- a/Makefile.pre.in
+++ b/Makefile.pre.in
@@ -2857,6 +2857,10 @@ python-config: $(srcdir)/Misc/python-config.in Misc/python-config.sh
 	@if test `uname -s` = Darwin; then \
 		cp python-config.py python-config; \
 	fi
+	@ # On Windows, use the python version of the script so no bash is required
+	@if test "$(MACHDEP)" = "win32"; then \
+		cp python-config.py python-config; \
+	fi
 
 # macOS' make seems to ignore a dependency on a
 # "$(BUILD_SCRIPTS_DIR): $(MKDIR_P) $@" rule.
